src/main/java/io/kokuwa/maven/helm/pojo/HelmExecutable.java

```diff
@@ -32,7 +32,7 @@
 @RequiredArgsConstructor
 public class HelmExecutable {
 
-	private final Log log;
+	public final Log log;
```

Comment:
Q: Why would any other class use the internal logger of `HelmExecutable`?

Reply:
I thought I fixed this back to private, this was an accident